package.json

```diff
@@ -2,10 +2,12 @@
   "name": "alpha-blog",
   "private": true,
   "dependencies": {
+    "@popperjs/core": "^2.11.5",
     "@rails/actioncable": "^6.1.5",
     "@rails/activestorage": "^6.1.5",
     "@rails/ujs": "^6.1.5",
     "@rails/webpacker": "5.4.3",
+    "bootstrap": "^5.1.3",
     "turbolinks": "^5.2.0",
     "webpack": "^4.46.0",
     "webpack-cli": "^3.3.12"
```

Comment:
I searched around and found that the newly created v7 app and upgraded to v7 app has differences in `package.json`, specifically the `script`, you can check it [here](https://github.com/rails/rails/blob/main/actioncable/package.json), why is it like that?

Reply:
What are the differences?

Reply:
Something similar to this, with the `script` stating the `build: css` etc, [stackoverflow](https://stackoverflow.com/questions/70482086/rails-7-0-esbuild-running-app-gives-error-command-build-not-found)
```
{
  "name": "app",
  "private": "true",
  "dependencies": {
    "@hotwired/stimulus": "^3.0.1",
    ...
  },
  "scripts": {
    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds",
    "build:css": "tailwindcss -i ./app/assets/stylesheets/application.tailwind.css -o ./app/assets/builds/application.css"
  }
```

Reply:
do you know what "scripts" are?

Reply:
No, I think, have heard of it but not familiar

Reply:
anything on "scripts" mean you can do: `npm run <script name>`.
so, with the example you give above, you can do:
```
npm run build:css
```

Reply:
So, if you upgrade from v6, you will have v6 config. while v7 has v7 configs